modules/nested-tabs-html/assets/scss/frontend.scss

```diff
@@ -1,6 +1,351 @@
+				order: var(--n-tabs-icon-order);
+				overflow: hidden;
+
+				i {
+					font-size: var(--n-tabs-icon-size, var(--n-tabs-title-font-size));
+				}
+
+				svg {
+					width: var(--n-tabs-icon-size, var(--n-tabs-title-font-size));
+					height: var(--n-tabs-icon-size, var(--n-tabs-title-font-size));
+				}
+
+				&:empty {
+					display: none;
+				}
+			}
+
+			&[aria-selected="false"] {
+				background-color: var(--n-tabs-title-background-color);
+
+				&,
+				& a {
+					color: var(--n-tabs-title-color);
+				}
+
+				.e-n-tab-icon {
+
+					i {
+						color: var(--n-tabs-icon-color);
+					}
+
+					svg {
+						fill: var(--n-tabs-icon-color);
+					}
+
+					i,
+					svg {
+
+						&:last-child {
+							transform: translate(0, -100vh);
+							height: 0;
+							opacity: 0;
+						}
+					}
+				}
+			}
+
+			&[aria-selected="true"] {
+
+				&,
+				& a {
+					color: var(--n-tabs-title-color-active);
+				}
+
+				.e-n-tab-icon {
+
+					i {
+						color: var(--n-tabs-icon-color-active);
+					}
+
+					svg {
+						fill: var(--n-tabs-icon-color-active);
+					}
+
+					i,
+					svg {
+
+						&:first-child {
+							transform: translate(0, -100vh);
+							height: 0;
+							opacity: 0;
+						}
+					}
+				}
+
+				// Remove the hover animation from the active tabs.
+				&[class*="elementor-animation-"] {
+					&:focus,
+					&:active,
+					&:hover {
+						transform: initial;
+						animation: initial;
+					}
+				}
+			}
+		}
+	}
+
+	[data-layout="tabs"] {
+
+		.e-n-tab-title[aria-selected="false"]:hover {
+
+			&,
+			& a {
+				color: var(--n-tabs-title-color-hover);
+			}
+
+			.e-n-tab-icon {
+
+				i {
+					color: var(--n-tabs-icon-color-hover);
+				}
+
+				svg {
+					fill: var(--n-tabs-icon-color-hover);
+				}
+			}
+		}
+	}
+}
+
+.elementor .elementor-element.elementor-widget-n-tabs-html > .elementor-widget-container > .e-n-tabs[data-layout="tabs"] > .e-n-tabs-heading .e-n-tab-title[aria-selected="false"]:hover {
+	background-color: var(--n-tabs-title-background-color-hover);
+	background-image: initial;
+}
+
+// TODO: Fix `Group_Control_Background` control. Currently the global default colours aren't generated inside CSS.
+.elementor .elementor-element.elementor-widget-n-tabs-html > .elementor-widget-container > .e-n-tabs > .e-n-tabs-heading .e-n-tab-title {
+
+	&[aria-selected="true"] {
+		background-color: var(--n-tabs-title-background-color-active);
+		background-image: initial;
+	}
+}
+
+// Responsive mode
+@mixin accordion-view-($device) {
+	.elementor.elementor .elementor-widget-n-tabs-html.e-n-tabs-#{$device} {
+		--n-tabs-direction: column;
+		--n-tabs-heading-display: contents;
+		--n-tabs-content-display: contents;
+
+		.e-n-tabs {
+			gap: 0;
+
+			&-content > .e-con {
+				order: var(--n-tabs-title-order);
+
+				&:not(.e-active) {
+					display: none;
+				}
+			}
+		}
+
+		.e-n-tab-title {
+			order: var(--n-tabs-title-order);
+
+			&:not(:first-child) {
+				margin-top: var(--n-tabs-title-gap);
```

Comment:
@rami-elementor Are we sticking to 'margin-top' for now?

Reply:
No, I already replaced all the margins to logical properties. Please replace `margin-top` with `margin-block-start`.

And also see the following PRs:
- https://github.com/elementor/elementor/pull/22524/files
- https://github.com/elementor/elementor/pull/22068/files
- https://github.com/elementor/elementor/pull/22989/files
- (and the list goes on)

While the widget is an experimental feature, you need to replace all the CSS to logical properties. When it will be a stable feature, we won't be able to fix it.

Reply:
Update